client/blocks/reader-full-post/style.scss

```diff
@@ -412,6 +412,24 @@
 	margin: 28px 0 26px;
 }
 
+.reader-full-post .embed-youtube,
+.reader-full-post .embed-vimeo {
+	display: block;
+	position: relative;
+	padding: 25px 0 56.25%;
+
+	iframe {
+		height: 100%;
```

Comment:
forgive the possible ignorance, but how does this work? isn't this saying "make my height 100% of my container"?